src/core/inline-idl-parser.js

```diff
@@ -72,18 +73,19 @@ function parseInlineIDL(str) {
 
 function renderBase(details) {
   // Check if base is a local variable in a section
-  const { identifier } = details;
+  const { identifier, renderParent } = details;
+  if (!renderParent) return document.createDocumentFragment();
```

Comment:
Maybe should just return without a value?

Reply:
hyperhtml shouts at
```
render`<code>${output}</code>`
```
if `output` contains `undefined`

Reply:
We still can return without value, and then do not `.push()` when caller gets nothing.